algebird-test/src/test/scala/com/twitter/algebird/MaxLaws.scala

```diff
@@ -29,6 +29,13 @@ class MaxLaws extends CheckProperties {
     }
   }
 
+  property("Max.semigroup returns the maximum item") {
+    forAll { v: NonEmptyVector[Int] =>
+      val maxItems = v.items.map { Max(_) }
+      v.sorted.last == Max.semigroup[Int].combineAllOption(maxItems).get.get
```

Comment:
Can you also test this for a custom type that does not have a special instance defined?

Also, scala has .max on Iterable. Can we just use that rather than sorted.last?

Reply:
Done.